### PR TITLE
188 integrate on ramp on the portal

### LINKED ISSUE #188: Integrate on-ramp on the portal.
As portal user, i should be able to buy PEN from the portal

## Acceptance criteria
* Show `Buy PEN` on the navigation menu
* On `\buy` do a page integration of the on-ramp UI
* Details to a page integration - https://alchemypay.readme.io/docs/page-integration-2 
* Integration approach should be redirection
* This option should only be available on the Pendulum version of Portal

### PR DESCRIPTION
**Pull request by gonzamontiel**
You'll notice that I have added a prodUrl and a testUrl for alchemy pay.
The testUrl is not being used in the NavBar, since tha last minute requirement was to show the link only when connected to Pendulum. I still decided to leave the testUrl (which includes a test appId) just in case we need it in the future.

Closes #188

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -47,4 +47,8 @@ export const config = {
     url: 'wss://relay.walletconnect.com',
     projectId: '299fda67fbf3b60a31ba8695524534cd',
   },
+  aclhemyPay: {
```
Comment (ebma) [inline src/config/index.ts:50]: Nit: typo

**Review by ebma — COMMENTED** [review]
As is mentioned in the info box of the [docs for on-ramping](https://alchemypay.readme.io/docs/page-integration-2), I'd recommend we add a `redirectUrl` parameter to the URL so that users get redirected to the portal once done.

File: src/config/index.ts (hunk 1 above)
Comment (gonzamontiel) [reply src/config/index.ts]: I might need a pair of glasses :eyes:  :eyeglasses:

**Comment by gonzamontiel** [comment]
> As is mentioned in the info box of the [docs for on-ramping](https://alchemypay.readme.io/docs/page-integration-2), I'd recommend we add a `redirectUrl` parameter to the URL so that users get redirected to the portal once done.

Sure, I've included it but I didn't test it (I would need some cash to buy PEN tokens), this can be done by product maybe. @prayagd ?

**Review by ebma — APPROVED** [review]
Looks good. @prayagd do you want to QA this by going through the whole process before we merge?

**Comment by prayagd** [comment]
@ebma @gonzamontiel Sure, Can do the QA.

**Comment by gonzamontiel** [comment]
> @ebma @gonzamontiel Sure, Can do the QA.

I read your comments and saw everything green so I merged. If the QA is still pending I recommend doing so on staging, and if anything, creating a bug ticket.

**Pull request merged**